tests/generator/test_make_fp.py

```diff
@@ -810,6 +821,7 @@ def test_make_fp_pwmat(self):
         atom_types = [0, 1, 2, 2, 0, 1]
         type_map = jdata['type_map']
         _make_fake_md(0, md_descript, atom_types, type_map)
+        print("Anguse : debug pwmat ", jdata)
```

Comment:
Remove the printing or use logging?

Reply:
Fixed. I had to check this since the unittest for DP-GEN cannot pass on my MacOS, and I figured out that it's because in PWMAT interface's  unittest, a binary script https://github.com/deepmodeling/dpgen/blob/devel/tests/generator/out_data_post_fp_pwmat/02.fp/task.000.000000/poscar2config.x  was involved and could only execute on Linux.

Reply:
This script was used to convert a POSCAR to atom's configuration file in PWMAT's format. I think it should be replaced with dpdata's functions.